### PR TITLE
Support djangocms-static-ace to load ace code editor from static files

### PR DESCRIPTION
**Pull request by fsbraun**
## Description

This PR allows the code editor ace to be served from static files with the help of [djangocms-static-ace](https://github.com/django-cms/djangocms-static-ace).

En passant add color scheme detection for the ace editor (dark mode).

## Related resources

* Fixes this issue: https://github.com/django-cms/djangocms-frontend/issues/48 .

## Checklist

* [x] I have opened this pull request against ``master``
* [x] I have added or modified the tests when changing logic
* [x] I have followed [the conventional commits guidelines](https://www.conventionalcommits.org/) to add meaningful information into the changelog
* [x] I have read the [contribution guidelines ](https://github.com/django-cms/django-cms/blob/develop/CONTRIBUTING.rst) and I have joined #workgroup-pr-review on 
[Slack](https://www.django-cms.org/slack) to find a “pr review buddy” who is going to review my pull request.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -83,6 +83,11 @@ For a manual install:
 
 * run ``python manage.py migrate``
 
+The Code plugin uses the ace code editor which is loaded from a CDN by default.
+If you want the ace code editor to be served from static files, please use
+`djangocms-bootstrap4[static-ace]` instead of `djangocms-bootstrap4` in your
```
Comment (marksweb) [inline README.rst]: @fsbraun These should be double back-quotes for code formatting.

https://docutils.sourceforge.io/docs/user/rst/quickstart.html#text-styles

**Review by marksweb — CHANGES_REQUESTED** [review]

**Comment by marksweb** [comment]
@fsbraun based on the failing test, this may also need an `apps.py` to define the default auto field.

**Comment by fsbraun** [comment]
@marksweb The new migrations seem to be happening for cms 3.9. Not sure, `apps.py` will help. Probably cms 3.9 needed to have a default auto field defined?!?

File: README.rst (hunk 1 above)
Comment (fsbraun) [reply README.rst]: Done.

**Comment by fsbraun** [comment]
I now skip the migration test for django cms 3.9. This will probably be an issue for other apps, too.

**Review by robert-foreflight — APPROVED** [review]

**Comment by fsbraun** [comment]
@marksweb If you're fine with this, I'd love to merge it and then #145

**Comment by marksweb** [comment]
@fsbraun I'll take a look shortly

**Review by marksweb — APPROVED** [review]

**Pull request merged**